config/prow/cluster/BUILD.bazel

```diff
@@ -57,6 +57,11 @@ release(
         "crd",
         cluster = BUILD_CLUSTER,
     ),
+    component(
+        "build_kubernetes-external-secrets",
```

Comment:
This overlaps with the rule right above, you'll need to name this something different, `build_kubernetes-external-secrets_customresource` maybe? Also rename the yaml file